setup/install/providers/aws/aws-ecs.md

```diff
@@ -1,9 +1,8 @@
 ---
 layout: single
-title:  "ECS (Amazon Web Services)"
+title:  "Amazon ECS"
 sidebar:
   nav: setup
-redirect_from: /setup/providers/ecs/
```

Comment:
I'd keep this redirect_from, _and_ add another one from `/setup/install/providers/ecs/`